mlrun/feature_store/api.py

```diff
@@ -175,6 +176,14 @@ def get_offline_features(
                                     By default, the filter executes on the timestamp_key of each feature set.
                                     Note: the time filtering is performed on each feature set before the
                                     merge process using start_time and end_time params.
+    :param additional_filters: (list of tuples, optional): List of additional_filters conditions as tuples.
```

Comment:
Would be nice to follow the convention of how we specify type and optionality in the rest of the code base. I guess there isn't exactly one way that we do this, but better not introduce one more way. Docs should be consistent.